lib/streamlit/config.py

```diff
@@ -444,32 +444,6 @@ def _global_development_mode() -> bool:
     type_=int,
 )
 
-_create_option(
-    "global.storeCachedForwardMessagesInMemory",
-    description="""
-        If True, store cached ForwardMsgs in backend memory. This is an
-        internal flag to validate a potential removal of the in-memory
-        forward message cache.
-    """,
-    visibility="hidden",
-    default_val=True,
-    type_=bool,
-)
-
-_create_option(
-    "global.includeFragmentRunsInForwardMessageCacheCount",
```

Comment:
When we remove this option but someone else sets it in their code, does it throw an error or is it simply a no-op?

Reply:
Users will just see a warning in the logs that they used an unknown config option.